### ISSUE TITLE
docker mount error

### ISSUE THREAD

**Issue by timchenxiaoyu** [moby/moby#35338]
docker version 12.6,docker run can mount /var/lib/docker but can not mount 
/var/lib/docker/containers

[root@master ~]# docker run -v /var/lib/docker:/tmp busybox   ls /tmp/
containers
devicemapper
image
network
swarm
tmp
trust
volumes
[root@master ~]# 
[root@master ~]# docker run -v /var/lib/docker/containers:/tmp busybox   ls /tmp/
[root@master ~]# 
[root@master ~]#

**Comment by ripcurld0** [comment]
@timchenxiaoyu can you provide more details like `docker info` and `docker version`.
Thanks

cc @thaJeztah @cpuguy83

**Comment by thaJeztah** [comment]
This works fine for me, but it looks like you're running a really outdated version of docker;

Possibly access is blocked by SELinux; try if you are able to access by adding `--security-opt="label=disable"`

Be aware that anything inside `/var/lib/docker/` is designed to be exclusively accessed by the docker daemon; interacting with files and directories inside that directory is highly discouraged and can result in problems (and even data corruption).

Please keep in mind that the GitHub issue tracker is not intended as a general support forum,
but for [reporting bugs and feature requests](https://raw.githubusercontent.com/moby/moby/master/.github/ISSUE_TEMPLATE.md). For other type of questions, consider using one of;

- the Docker Support Forums - https://forums.docker.com
- the [Docker community Slack channel](https://blog.docker.com/2016/11/introducing-docker-community-directory-docker-community-slack/) (register here: [http://dockr.ly/community](http://dockr.ly/community))
- [StackOverflow](https://stackoverflow.com)

If you think this is a bug, and you're able to reproduce on a current version of docker, please open a new issue and provide the information that's requested in the [issue-template](https://raw.githubusercontent.com/docker/docker/master/.github/ISSUE_TEMPLATE.md) that was shown when you opened this issue.

- provide the output of `docker version`
- provide the output of `docker info`
- provide the exact steps to reproduce. If the steps to reproduce require custom images, or source code, provide a _minimal_ test-case to reproduce. Keep in mind that we don't have access to your private images or source code. Without steps to reproduce, or a reproducible case it's unlikely that your issue can be resolved.


I'm closing this issue because this is not a bug, but feel free to continue the conversation 👍

**Comment by timchenxiaoyu** [comment]
docker info
Containers: 88
 Running: 30
 Paused: 0
 Stopped: 58
Images: 25
Server Version: 1.12.6
Storage Driver: devicemapper
 Pool Name: docker-253:1-394733-pool
 Pool Blocksize: 65.54 kB
 Base Device Size: 10.74 GB
 Backing Filesystem: xfs
 Data file: /dev/loop0
 Metadata file: /dev/loop1
 Data Space Used: 7.13 GB
 Data Space Total: 107.4 GB
 Data Space Available: 10.66 GB
 Metadata Space Used: 10.99 MB
 Metadata Space Total: 2.147 GB
 Metadata Space Available: 2.136 GB
 Thin Pool Minimum Free Space: 10.74 GB
 Udev Sync Supported: true
 Deferred Removal Enabled: true
 Deferred Deletion Enabled: true
 Deferred Deleted Device Count: 0
 Data loop file: /var/lib/docker/devicemapper/devicemapper/data
 WARNING: Usage of loopback devices is strongly discouraged for production use. Use `--storage-opt dm.thinpooldev` to specify a custom block storage device.
 Metadata loop file: /var/lib/docker/devicemapper/devicemapper/metadata
 Library Version: 1.02.140-RHEL7 (2017-05-03)
Logging Driver: json-file
Cgroup Driver: systemd
Plugins:
 Volume: local
 Network: host null bridge overlay
Swarm: inactive
Runtimes: docker-runc runc
Default Runtime: docker-runc
Security Options: seccomp
Kernel Version: 3.10.0-327.4.5.el7.x86_64
Operating System: CentOS Linux 7 (Core)
OSType: linux
Architecture: x86_64
Number of Docker Hooks: 3
CPUs: 8
Total Memory: 15.51 GiB
Name: master
ID: W7OO:HFPL:WSDE:5WU3:LMME:Y3WG:5DQA:KETP:OEHC:W4DS:SVSA:DHIC
Docker Root Dir: /var/lib/docker
Debug Mode (client): false
Debug Mode (server): false
Registry: https://index.docker.io/v1/
WARNING: bridge-nf-call-ip6tables is disabled
Insecure Registries:
 127.0.0.0/8
Registries: docker.io (secure)



[root@master ~]# docker version
Client:
 Version:         1.12.6
 API version:     1.24
 Package version: docker-1.12.6-48.git0fdc778.el7.centos.x86_64
 Go version:      go1.8.3
 Git commit:      0fdc778/1.12.6
 Built:           Thu Sep  7 18:00:07 2017
 OS/Arch:         linux/amd64

Server:
 Version:         1.12.6
 API version:     1.24
 Package version: docker-1.12.6-48.git0fdc778.el7.centos.x86_64
 Go version:      go1.8.3
 Git commit:      0fdc778/1.12.6
 Built:           Thu Sep  7 18:00:07 2017
 OS/Arch:         linux/amd64

**Comment by thaJeztah** [comment]
It looks like you're running the Red Hat fork of Docker; are you seeing the same with the current version of the official packages? https://docs.docker.com/engine/installation/linux/docker-ce/centos/
